pkg/authorization/api/types.go

```diff
@@ -189,7 +189,7 @@ type ResourceAccessReview struct {
 	unversioned.TypeMeta
 
 	// Action describes the action being tested
-	Action AuthorizationAttributes
+	Action
```

Comment:
anonymous inclusion makes me sad.  We can't get autogeneration and keep this named?